include/engine/engine.hpp

```diff
@@ -20,6 +18,9 @@
 #include "engine/status.hpp"
 #include "util/json_container.hpp"
 
+#include "engine/data_watchdog.hpp"
```

Comment:
This is actually the workaround: interprocess headers must be included after Boost.Range. It is somehow related to inclusion of /usr/include/boost/interprocess/detail/workaround.hpp

If someone can update boost in osrm-deps-win-x64-14.0.7z  to 1.61 or 1.62 probably it will not be required.